lldb/source/Symbol/Type.cpp

```diff
@@ -75,20 +75,18 @@ bool lldb_private::contextMatches(llvm::ArrayRef<CompilerContext> context_chain,
 static CompilerContextKind ConvertTypeClass(lldb::TypeClass type_class) {
   if (type_class == eTypeClassAny)
     return CompilerContextKind::AnyType;
-  uint16_t result = 0;
-  if (type_class & lldb::eTypeClassClass)
-    result |= (uint16_t)CompilerContextKind::Class;
-  if (type_class & lldb::eTypeClassStruct)
-    result |= (uint16_t)CompilerContextKind::Struct;
```

Comment:
This is a potentially controversial change. It means that if someone does a `image lookup -t 'struct Foo'`, we will also return a `class Foo`. This could be viewed as confusing, or as consistent with how we handle structures/classes elsewhere...

Reply:
Hmm it might actually be useful to see both in the output?

Reply:
> Hmm it might actually be useful to see both in the output I would've thought?

Maybe, I don't really have an intuitive expectation here, but I like how this forces a consistent behavior. (There are other ways to ensure a consistent struct/class handling, but they would require more code.)